docs/getting-started/development.md

```diff
@@ -4,12 +4,16 @@ In development environments, Ubiquitous servers are created and managed in Virtu
 
 ## Getting started
 
-First, prepare your development by installing the following applications:
+### Prerequisites
+
+Install and be familiar with:
 
 * [VirtualBox](https://www.virtualbox.org/) --- desktop virtualisation
 * [Vagrant](https://www.vagrantup.com/) --- command line tool for managing virtualised development environments
 
-Then install some Vagrant plugins that'll make it easier to manage larger environments:
+Also be familiar with Salt - see [Salt Admin](../roles/salt)
```

Comment:
This link will be broken -- GItBook links have to be to the markdown file, e.g. `../roles/salt.md` in this case. I don't think this document is useful for familiarising yourself with Salt though. The links [here](https://github.com/LukeCarrier/moodle-ubiquitous/blob/master/docs/README.md#developing-ubiquitous) are probably more what we're looking for?